platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileCommon.java

```diff
@@ -58,6 +63,19 @@ public final class DataFileCommon {
 
     private static final Logger logger = LogManager.getLogger(DataFileCommon.class);
 
+    /** Access to sun.misc.Unsafe required for atomic compareAndSwapLong on off-heap memory */
```

Comment:
This is a bit misleading, as this class uses UNSAFE to clean up memory mapped buffers, not compare and swap long.

Reply:
This code is just moved from `DataFileWriter` to `DataFileCommon` (as it will be used in `DataFileReader`, too) along with the comment. I'll update the comment in some future changesets in this branch to

```java
    /**
     * Access to sun.misc.Unsafe required to close mapped byte buffers explicitly rather than
     * to rely on GC to collect them.
     */
```